apps/shop/tests.py

```diff
@@ -6,34 +9,168 @@
 from apps.authentication.models import Email, OnlineUser
 from apps.inventory.models import Item
 from apps.oauth2_provider.test import OAuth2TestCase
+from apps.payment.models import PaymentTransaction
+from apps.payment.transaction_constants import TransactionSource
 from apps.shop.models import MagicToken
 
 
-class ShopAPIURLTestCase(APITestCase):
-    def test_item_list_empty(self):
-        url = reverse("item-list")
+class ShopItemTestCase(APITestCase):
+    basename = "shop_inventory"
+
+    def get_list_url(self):
+        return reverse(f"{self.basename}-list")
 
-        response = self.client.get(url)
+    def get_detail_url(self, _id):
```

Comment:
There might be a naming-conflict I'm not seeing, or some Python/Django-way of doing things, but why is this marked as unused?

Reply:
Not seeing it :man_shrugging:

Reply:
By unused, I mean the `_name` naming, which is usually used if you want arguments that are perhaps unused, I was just wondering why it is named that way?

Reply:
The `_name` convention is used for private methods and fields